reflex/reflex.py

```diff
@@ -255,6 +269,38 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
+    # explicitly running `reflex db migrate` overrides config file value
+    get_config().db_migrate = True
+    prerequisites.get_app()
+    prerequisites.check_schema_up_to_date()
+
+
+@db_cli.command()
+def makemigrations():
+    """Create autogenerated alembic migration scripts."""
+    if not model.has_alembic:
+        console.print(
+            "[red]Alembic is not installed. Run [bold]pip install alembic[/bold] first."
+        )
+        return
+    if not Path(constants.ALEMBIC_CONFIG).exists():
+        console.print(
+            "[red]Alembic is not initialized. Run [bold]python -m alembic init alembic[/bold] first."
+        )
```

Comment:
Is there a specific reason for not putting this behind a `reflex db init`?

Reply:
also a great idea, and that's been done in the latest commit.

if you're using the db built into reflex, you will be using alembic. otherwise you can bring your own database, since the db layer has been pulled out into its own CLI and never does migrations by default anymore.